packages/shared/lib/services/sync/on-event-scripts.service.ts

```diff
@@ -19,39 +28,35 @@ export const onEventScriptService = {
     }): Promise<void> {
         await db.knex.transaction(async (trx) => {
             const onEventInserts: Omit<OnEventScript, 'id' | 'created_at' | 'updated_at'>[] = [];
+
+            // Deactivate all previous scripts for the environment
+            // This is done to ensure that we don't have any orphaned scripts when they are removed from nango.yaml
+            const previousScriptVersions = await trx
+                .from<OnEventScript>(TABLE)
+                .whereRaw(`config_id IN (SELECT id FROM _nango_configs WHERE environment_id = ?)`, [environment.id])
```

Comment:
didn't check but could there be an issue if we only deploy one provider/script?

Reply:
good point. I missed that